style.css

```diff
@@ -0,0 +1,229 @@
+    background: #003147;
+    padding: 40px;
+}
+.profiletext{
+    position: relative;
+
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    padding-bottom: 20px;
+    border-bottom: 1px solid rgba(255,255,255,0.2);
+}
+.profiletext .imgbox{
+    position: relative;
+
+    width: 200px;
+    height: 200px;
+
+    border-radius: 50%;
+
+    overflow: hidden;
+
+}
+.profiletext .imgbox img{
+    position: absolute;
+
+    inset: 0;
+
+    width: 100%;
+    height: 100%;
+
+    object-fit: cover;
+}
+.profiletext h2{
+    color: #fff;
+
+    font-size: 1.8em;
+    font-weight: 600;
+
+    margin-top: 10px;
+
+    text-transform: uppercase;
+    text-align: center;
+
+    line-height: 1.4em;
+}
+.profiletext h3{
+    color: #fff;
+    font-size: 1.3em;
+    font-weight: 300;   
+
+    margin-top: 5px;
+}
+.contactinfo{
+    padding-top: 40px;
+}
+.title{
+    color: #fff;
+    text-transform: uppercase;
+    font-weight: 600;
+    letter-spacing: 1px;
+    margin-bottom: 20px;
+}
+.contactinfo ul{
+    position: relative;
+}
+.contactinfo ul li{
+    position: relative;
+    list-style-type: none;
+
+    margin: 10px 0;
+    cursor: pointer;
+}
+.contactinfo .icon{
+    display: inline-block;
+    width: 30px;
+    font-size: 18px;
+    color: #03a9f4;
+}
+.contactinfo ul li span {
+    color: #fff;
+    font-weight: 300;
+}
+.contactinfo .education li{
+    margin: 15px;
+}
+.contactinfo h4{
+    color: #03a9f4;
+    font-weight: 500;
+}
+.contactinfo h5{
+    color: #fff;
+
+    font-weight: 500;
+    font-size: 16px;
+
+    letter-spacing: 2px;
+}
+.contactinfo.language .percent{
+    position: absolute;
+    
+    width: 100%;
+    height: 6px;
+    margin-top: 2px;
+
+    background: #081921;
+
+    display: block;
+}
+.contactinfo.language .percent div{
+    position: absolute;
+
+    inset: 0;
+
+    height: 100%;
+    background: #03a9f4;
+}
+/* Right side*/
+.container  .right_side{
+    position: relative;
+    background: #fff;
+    padding: 40px;
+}
+.about{
+    margin-bottom: 50px;
+}
+.title2{
+    color: #003147;
+    text-transform: uppercase;
+
+    letter-spacing: 1px;
+    margin-bottom: 10px;
+    font-weight: 500;
+}
+p{
+    color: #333;
+}
+.about .box h4{
+    margin-bottom: 10px;
+    font-size: 20px;
+}
+.about .box p{
+    position: relative;
+}
+.skills .box h4{
+    text-transform: uppercase;
+
+    color: #848c99;
+
+    font-weight: 500;
+    
+}
+.skills .box .percent{
+    position: relative;
+
+    width: 100%;
+    height: 10px;
+
+    background: #f0f0f0;
+}
+.skills .box .percent div{
+    position: absolute;
+
+    inset: 0;
+
+    height: 100%;
+
+    background: #03a9f4;
+}
+.interest ul{
+    display: grid;
+    grid-template-columns: repeat(3,1fr);
+}
+.interest ul li{
+    list-style: none;
+
+    color: #333;
+    font-weight: 500;
+
+    margin: 20px 0;
+}
+.interest ul li .fa{
+    color: #09a3f4;
+
+    font-size: 20px;
+    width: 30px;
+}
+@media (max-width: 1000px){
```

Comment:
declare media type ??

Reply:
plzz tell me how to declare media types?